config/data/drugs.yml

```diff
@@ -0,0 +1,1025 @@
+---
+- id: e9865df7-39b5-4fb8-8ab3-354f87a5d369
+  name: Amlodipine
+  composition:
```

Comment:
Double checking, do we want to render these empty fields as `nil` or empty string in the JSON?

Reply:
Its a nullable string, so nil is better